libs/utils/android/workload.py

```diff
@@ -134,4 +134,31 @@ def tracingStop(self):
         # Dump a platform description
         self._te.platform_dump(self.out_dir)
 
+    def traceShow(self):
+        """
+        Open the collected trace using the most appropriate native viewer.
+
+        The native viewer depends on the specified trace format:
+        - ftrace: open using kernelshark
+        - systrace: open using a browser
+
+        In both cases the native viewer is assumed to be available in the host
+        machine.
+        """
+
+        if 'ftrace' in self.collect:
+            os.popen("kernelshark {}".format(self.trace_file))
+            return
+
+        if 'systrace' in self.collect:
```

Comment:
I've never used it before, but maybe https://docs.python.org/2/library/webbrowser.html would be worth looking into ? It does add yet another module to LISA, but it makes it so that we don't have to manually look at the current OS.